### DISCUSSION TITLE
App rejected by Apple: Missing purpose string in your app’s Info.plist for Motion & Fitness Activity information

### DISCUSSION THREAD

**Discussion by siticum** [buildnatively/community#69] [Ideas]
### Use case

```Text
We noticed that your app requests the user’s consent to access the Motion & Fitness Activity information, but doesn’t sufficiently explain the use of the Motion & Fitness Activity information in the purpose string.

To help users make informed decisions about how their data is used, permission request alerts need to explain and include an example of how your app will use the requested information.

Next Steps

Please revise the purpose string in your app’s Info.plist file for the Motion & Fitness Activity information to explain why your app needs access and include an example of how the user's data will be used.

You can modify your app's Info.plist file using the property list editor in Xcode.
```

### Idea

```Text
Add a user input for purpose of using Motion & Fitness Activity information in the Natively Dashboard.
```

**Comment by siticum** [comment]
![image](https://github.com/buildnatively/community/assets/151831088/6018ba68-6bbf-4622-8744-e90ab208f557)

**Comment by romanfurman6** [comment]
Hey @siticum 

I apologize for the delay in responding to your message. Our team was taking some time off for the 2024 holiday season.

Thank you for submitting an issue. We suggest following these steps to resolve your request from Apple: https://github.com/gree/unity-webview/issues/933#issuecomment-1587308353.

**Comment by siticum** [comment]
Hi Roman

Thanks. I found a way around it as well, and got the app approved!

Disable the request action > submit and get it approved > re-enable the
action -Shhhh,  don’t tell anyone 😅

Strange practice they are running, not allowing changes to the message but
rejection apps on that basis 🙃

**Reply by romanfurman6** [reply]
haha )) ye this should also work :D
